src/Microsoft.Extensions.FileProviders.Physical/PhysicalFileProvider.cs

```diff
@@ -147,11 +171,24 @@ public IFileInfo GetFileInfo(string subpath)
         /// </summary>
         /// <param name="subpath">A path under the root directory. Leading slashes are ignored.</param>
         /// <returns>
-        /// Contents of the directory. Caller must check Exists property. <see cref="NotFoundDirectoryContents" /> if
+        /// Contents of the directory. Caller must check <see cref="IDirectoryContents.Exists"/> property. <see cref="NotFoundDirectoryContents" /> if
         /// <paramref name="subpath" /> is absolute, if the directory does not exist, or <paramref name="subpath" /> has invalid
         /// characters.
         /// </returns>
         public IDirectoryContents GetDirectoryContents(string subpath)
+            => GetDirectoryContents(subpath, _excludedFiles);
+
+        /// <summary>
+        /// Enumerate a directory at the given path, if any.
+        /// </summary>
+        /// <param name="subpath">A path under the root directory. Leading slashes are ignored.</param>
+        /// <param name="excludedFiles">Specifies which files will be excluded from the results.</param>
+        /// <returns>
+        /// Contents of the directory. Caller must check <see cref="IDirectoryContents.Exists"/> property. <see cref="NotFoundDirectoryContents" /> if
+        /// <paramref name="subpath" /> is absolute, if the directory does not exist, or <paramref name="subpath" /> has invalid
+        /// characters.
+        /// </returns>
+        public IDirectoryContents GetDirectoryContents(string subpath, ExcludedFiles excludedFiles)
```

Comment:
Same reason for adding the GetFileInfo overload. I'd like to use the default file provider most of the time, but still be able to bypass its filtering in special cases.